charts/prometheus-blackbox-exporter/templates/servicemonitor.yaml

```diff
@@ -16,7 +16,15 @@ metadata:
 spec:
   endpoints:
   - port: http
-    scheme: http
+    {{- if .Values.serviceMonitor.scheme }}
+    scheme: {{ .Values.serviceMonitor.scheme }}
```

Comment:
this breaks current behaviour as the value would change.
imho this would make this a breaking change.

Reply:
Good catch !
Changed the default values to http so I can keep changing the scheme in case I'm doing authenticated prometheus (with oAuth proxy on OCP)